roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -108,6 +131,16 @@ class Interface(object):
 
         subprocess.check_call(cmd)
 
+    def attach(self, bridge=None):
+        if self.ifname.startswith('tap') \
```

Comment:
Attaching interfaces to a bridge is not restricted to tap/tun interfaces. Can we move this check out of this method?